modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -557,7 +557,8 @@ const register = (editor: Editor) => {
   });
 
   registerOption('element_format', {
-    processor: 'string'
+    processor: 'string',
+    default: 'html'
```

Comment:
This needs a few other changes sorry Luke, we'll also need to change the `Writer.ts` logic to default to `html` as atm it'll default to `xhtml`.

- [ ] Update the writer to default to HTML output as well